service/controller/chart/resource/status/create.go

```diff
@@ -115,3 +126,39 @@ func (r *Resource) setStatus(ctx context.Context, cr v1alpha1.Chart, status v1al
 
 	return nil
 }
+
+func updateEvent(webhookURL string, status v1alpha1.ChartStatus) error {
```

Comment:
```suggestion
func updateAppStatus(webhookURL string, status v1alpha1.ChartStatus) error {
```

I'd like the name to be clearer. WDYT to this?

Reply:
Much better!